### ISSUE TITLE
Error: Cannot compute gradient: gradient function not found for gather

### ISSUE THREAD

**Issue by zaidalyafeai** [tensorflow/tfjs#150]
#### TensorFlow.js version
"0.6.1"
#### Browser version
Chrome Version 65.0.3325.181 (Official Build) (64-bit)
#### Describe the problem or feature request
"I get the error Error: Cannot compute gradient: gradient function not found for gather" when calling the fit function on RNN model 
#### Code to reproduce the bug / link to feature request
` model = tf.sequential();
  model.add(tf.layers.embedding(
  {
     inputDim:VOCAB_SIZE,
     outputDim:EMBED_SIZE
  }));
  //model.add(tf.layers.lstm({units:16,returnSequences: true}));
  //model.add(tf.layers.lstm({units: 8,returnSequences: true}));
  model.add(tf.layers.simpleRNN ({units: 2}));

  //model.add(tf.layers.flatten());
  model.add(tf.layers.dense({
    units: 1,
    kernelInitializer: "VarianceScaling",
    activation: "sigmoid"
  }));
  
  //compile the model using and adam optimizer 
  model.compile({
    optimizer: tf.train.adam(0.001),
    loss: "binaryCrossentropy",
    metrics: ["accuracy"]
  });

  h = model.fit(tf.zeros([BATCH_SIZE,MAX_STRING_SIZE]), tf.zeros([BATCH_SIZE,1]))
}`

**Comment by ManrajGrover** [comment]
Hi @zaidalyafeai,

Thank you for reporting this. I believe this is because gradients for `tf.gather` are yet to be implemented and is being tracked in #135.

**Comment by nsthorat** [comment]
This is done.
